discord/opus.py

```diff
@@ -276,17 +276,14 @@ class _OpusStruct:
 
     @staticmethod
     def get_opus_version() -> str:
-        if not is_loaded():
-            if not _load_default():
-                raise OpusNotLoaded()
+        if not is_loaded() and not _load_default():
+            raise OpusNotLoaded()
 
         return _lib.opus_get_version_string().decode('utf-8')
 
 class Encoder(_OpusStruct):
     def __init__(self, application=APPLICATION_AUDIO):
-        if not is_loaded():
-            if not _load_default():
-                raise OpusNotLoaded()
+        _OpusStruct.get_opus_version()
```

Comment:
Just use `and`

Reply:
It's not necessary as get_opus_version checks this

Reply:
You could argue that this is now slower (and not optimized, going against the entire point of the PR) since get_opus_version calls another two methods unnecessarily.